src/Sentry.Unity.Editor/SentryWindow.cs

```diff
@@ -38,7 +50,28 @@ private void OnEnable()
                 AssetDatabase.CreateAsset(Options, SentryOptionsAssetPath);
             }
 
-            EditorUtility.SetDirty(Options);
+            EditorUtility.SetDirty(Options);*/
+        }
+
+        private UnitySentryOptionsJson LoadSentryJsonConfig()
+        {
+            if (!File.Exists(SentryOptionsJsonPathFull))
+            {
+                var unitySentryOptionsJson = new UnitySentryOptionsJson
+                {
+                    Dsn = "https://94677106febe46b88b9b9ae5efd18a00@o447951.ingest.sentry.io/5439417",

```

Comment:
We need to `#ifdef` or something so the test DSN only gets included in dev mode for the SDK. Meaning it shouldn't go into the unity package

Reply:
I don't think we discussed that yet. Possible point for discussion #83

Reply:
We can't have that hardcoded here, I'm aware we can't use unify's compiler flag since we ship the compiled DLLs. We could add a single script though that's on source control for us to use but it's not moved into the packages folders